tests/dv_backup_test.go

```diff
@@ -134,14 +134,15 @@ var _ = Describe("DV Backup", func() {
 			By("Creating clone DV - object under test")
 			dvSpec := framework.NewCloneDataVolume(dvName, "100Mi", srcDv.Namespace, srcDv.Name, f.StorageClass)
 			dv, err = framework.CreateDataVolumeFromDefinition(f.KvClient, f.Namespace.Name, dvSpec)
+			Expect(err).ToNot(HaveOccurred())
 
 			By("Creating backup test-backup")
 			err = framework.CreateBackupForNamespace(timeout, backupName, f.Namespace.Name, snapshotLocation, f.BackupNamespace, true)
 			Expect(err).ToNot(HaveOccurred())
 
 			phase, err := framework.GetBackupPhase(timeout, backupName, f.BackupNamespace)
 			Expect(err).ToNot(HaveOccurred())
-			Expect(phase).To(Equal(velerov1api.BackupPhaseCompleted))
+			Expect(phase).To(Equal(velerov1api.BackupPhasePartiallyFailed))
```

Comment:
This change is related to populators.  This test does a backup while clone in progress.  Velero does not like that the PVC is `Pending`.  I don't think there is anything we can do on our end to prevent this.